[CON-603] feat: enable Lemlist proxy

Jkarage opened the pull request:
## Testing
### GET

<img width="1152" alt="Screenshot 2024-12-02 at 16 33 28" src="https://github.com/user-attachments/assets/c4fd1b81-853f-4eb9-82bc-7dd8fdb06725">


### POST
<img width="1159" alt="Screenshot 2024-12-02 at 15 34 23" src="https://github.com/user-attachments/assets/1c928fe5-5ce3-415e-ab19-7adb828f2991">

### PATCH
<img width="1155" alt="Screenshot 2024-12-02 at 15 35 41" src="https://github.com/user-attachments/assets/6fd6d0bf-483b-49ca-8e70-43c614832aae">

### DELETE
<img width="1145" alt="Screenshot 2024-12-02 at 15 38 22" src="https://github.com/user-attachments/assets/9384fc2f-429b-4d80-a403-a535fc6f5a9b">


### Invalid requests
As observed while testing the API, Invalid request responds with 200 OK
<img width="1151" alt="Screenshot 2024-12-02 at 15 49 26" src="https://github.com/user-attachments/assets/5f9cd9ef-cdda-4a62-8299-4773388c2c89">



## Pagination
Pagination is only supported in the campaigns resource.
<img width="1163" alt="Screenshot 2024-12-02 at 15 37 17" src="https://github.com/user-attachments/assets/782c62bc-bbb8-4fa4-a027-61e5dd008356">

<img width="1280" alt="Screenshot 2024-12-02 at 15 37 48" src="https://github.com/user-attachments/assets/ebdeaca7-dd64-4eae-8c55-f5518408c0ee">


## Successful console operation (operation & events)
<img width="967" alt="Screenshot 2024-12-02 at 15 51 20" src="https://github.com/user-attachments/assets/a24c924e-2790-4922-8df2-68ca473e2e92">


## Failed console operation (operation & events)
<img width="970" alt="Screenshot 2024-12-02 at 15 52 33" src="https://github.com/user-attachments/assets/debee110-d959-4ad8-a822-3f5bcbcecaab">

RajatPawar reviewed (approved):
Can you please mention the 200 error code in the ticket as well? We might have to mention this in our docs. Also - if we use the API directly, does it still return a 200 for errors?

Jkarage commented:
Added it in ticket. Both the API and the Proxy responds the same. 200 for a not available/unknown resource. 404 for an available resource but no such record. I have added the behaviours in the slab as well as in the ticket.
